김윤건/4회차/week2./cousinsBT.js

```diff
@@ -0,0 +1,22 @@
+var isCousins = function(root, x, y) {
+  const queue = [root];
+
+  while (queue.length) {
+      if (queue.filter(node => node.val === x).length && queue.filter(node => node.val === y).length) return true;
```

Comment:
각 노드의 값은 unique한 값이므로 filter로 모든 배열을 비교하기 보다는 find을 이용해도 좋을 것 같네요!

```suggestion
      if (queue.find(node => node.val === x) && queue.find(node => node.val === y)) return true;
```